packages/mux-video-ads/src/index.ts

```diff
@@ -160,12 +161,29 @@ video::-webkit-media-text-track-container {
             this.#muxAdManager.requestAds(this.adTagUrl);
           }
         }
+        this.#isUsingSameVideoElement = this.#muxAdManager?.isUsingSameVideoElement();
       },
       { once: true }
     );
 
     this.addEventListener('play', this.play);
 
+    this.nativeEl.addEventListener('play', (event) => {
+      if (this.adBreak && !this.#isUsingSameVideoElement) {
+        console.warn('Video play prevented during ad break');
+        this.nativeEl.pause();
+        return;
+      }
+    });
+
+    this.nativeEl.addEventListener('seeking', (event) => {
+      if (this.adBreak && !this.#isUsingSameVideoElement) {
+        console.warn('Seek prevented during ad break');
+        this.nativeEl.currentTime = 0;
```

Comment:
this is fine for now/preroll, but will likely need to be a bit fancier for longer term